ovh/client.py

```diff
@@ -479,12 +479,17 @@ def raw_call(self, method, path, data=None, need_auth=True):
         :param str path: api entrypoint to call, relative to endpoint base path
         :param data: any json serializable data to send as request's body
         :param boolean need_auth: if False, bypass signature
+        :param dict headers: A dict containing the headers needing to be sent to
+                             the OVH API. The function may override the
```

Comment:
I'd suggest changing `may` to `will`, as it is not conditional, we *will* set them ourselves in the code

Reply:
Done